clang/test/CXX/drs/cwg27xx.cpp

```diff
@@ -134,7 +134,7 @@ void test() {
 }
 
 namespace cwg2798 { // cwg2798: 17
-#if __cpp_static_assert >= 202306
+#if __cplusplus > 202302L
```

Comment:
I guess this is one of the potentially-breaking changes that comes with changing the feature test macro. I think this is incorrect code and should have been `__cpp_static_assert >= 202306L && __cpp_constexpr >= 201907L`.

For DR testing, I don't think we check extensions, which is why I changed this to check `__cplusplus`. @Endilll is that right?

Reply:
Yeah, checking for `__cplusplus` makes more sense in DR test suite.
But I wonder if we can leverage feature test macros for a future conformance testing outside of DRs.